projboard/models/view.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from .article import Article
+from .user import User
+
+
```

Comment:
According to #40, #30 and #41 view class should be include in article.py and not in a separate file

Reply:
> According to #40, #30 and #41 view class should be include in article.py and not in a separate file

+1, to prevent conflicts

Reply:
done